webpack/webpack.config.dll.js

```diff
@@ -27,7 +27,6 @@ module.exports = {
       'react-notification-system',
       'react-portal-hoc',
       'react-redux',
-      'react-relay',
```

Comment:
How come?

Reply:
because relay is updating version much more frequently than i expected. it's a good thing because each version has awesome functionality (like the client schema) but it's frustrating to have it not work because you forgot to build the dll between branch switches

Reply:
If the tension behind this change has to do with development, you can add a git hook to do just that.

I'm just wondering if this will negatively effect production compilation.

Reply:
nah, DLL is only used in development (i use it to precompile 3rd party libs, we could also do it to precompile packages that won't be tested during the sprint, eg the welcome wizard module, but i just never took the time to figure that out... although it'd be pretty sweet to have ~500ms build times & instant refreshes)